### ISSUE TITLE
Cannot type in terminal when asked about authentication when cloning git repo

### ISSUE THREAD

**Issue by lisamariet** [microsoft/terminal#13772]
### Windows Terminal version

1.14.1962.0

### Windows build number

10.0.19044.1889

### Other Software

OpenSSH
GitForWindows

### Steps to reproduce

1. run command to clone a repository when first time authenticating with ssh key (I have tried both a bitbucket, and a github repo with same result)
git clone repoUrl + press enter

2. It askes if I want to continue connecting (yes/no/[fingerprint]) ?

### Expected Behavior

I should be able to type yes

### Actual Behavior

I cannot type anything. I can press _enter_ and it will complain that I need to type something... but no letter will type. 
- Ctrl + C does break me out of it and then I can type in the window again... 

<img width="493" alt="image" src="https://user-images.githubusercontent.com/5948299/185395421-54cb4104-dd24-4f2d-81c4-52d389b162ec.png">

**Comment by lisamariet** [comment]
Even stranger.. if I run command _ssh bitbucket.org_ It will allow me to type YES when asked the same question. 

<img width="575" alt="image" src="https://user-images.githubusercontent.com/5948299/185399791-08f90d3e-fefd-49f7-9026-27a86d1164e3.png">

**Comment by zadjii-msft** [comment]
Which version of `ssh`/?  Does this repro in both a Terminal window and a vintage console window (`conhost.exe`) the same way?

**Comment by lisamariet** [comment]
ssh -V reports:
OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2

In vintage console it behaves the same yes. 

When testing now. I realized that actually it does register that I type yes.. it just does not SHOW it in the console, like it does in powershell or bash shell.

So I typed yes and pressed enter and it did add the new location to known hosts and cloned the repo. So I guess this is fine then. I must have miss spelled 'yes' before.
